test/users.js

```diff
@@ -0,0 +1,67 @@
+const chai = require('chai'),
+  dictum = require('dictum.js'),
+  server = require('./../app'),
+  orm = require('../app/orm'),
+  should = chai.should();
+
+describe('/users POST', () => {
+  it('should fail because the password is too short', (done) => {
+    chai.request(server)
+      .post('/users')
+      .send({
+        firstName: 'xxxxxx', lastName: 'zzzzzzzzz',
+        password: '1234', email: '123456@wolox.com.ar'
+      })
+      .catch((err) => {
+        err.response.should.be.json;
+        err.response.body.should.have.property('error');
+        err.should.have.status(400);
+      })
+      .then(() => {
+        done();
+      });
+  });
+  it('should fail because something is missing', (done) => {
+    chai.request(server)
+      .post('/users')
+      .send({
+        firstName: 'xxxxxx',
+        password: '1234', email: '123456@wolox.com.ar'
+      })
+      .catch((err) => {
+        err.response.should.be.json;
+        err.response.body.should.have.property('error');
+        err.should.have.status(400);
+      })
+      .then(() => {
+        done();
+      });
+  });
+  it('should fail because of existing email', (done) => {
+    chai.request(server)
+      .post('/users')
+      .send({
+        firstName: 'xxxxxx', lastName: 'yyyyyyy',
+        password: '123456789', email: 'test@wolox.com.ar'
+      })
+      .catch((err) => {
+        err.response.should.be.json;
+        err.response.body.should.have.property('error');
+        err.should.have.status(500);
+      })
+      .then(() => done());
+  });
+  it('should be successful', (done) => {
+    chai.request(server)
+      .post('/users')
+      .send({
+        firstName: 'aaaaa', lastName: 'bbbbb',
+        password: '123456789', email: '12345@wolox.com.ar'
+      })
+      .then((res) => {
+        res.should.have.status(200);
```

Comment:
Here we could also check that we actually have created a new user, but for now its ok without checking.